modules/ROOT/pages/monitoring-dashboards.adoc

```diff
@@ -16,139 +17,142 @@ The app monitoring dashboard helps you answer questions about your integrated ap
+* *Breakdown*
++
+View detailed metrics that track each part of the server's memory separately.
 
-To view graphs for a specific time period, select the time interval on the top-right of the page.
+To view graphs for a specific period, select the time interval on the top-right of the page:
 
 image::monitoring-dashboards-cbd92.png[]
 
-Hover your mouse over a given point of a graph, to display an overlay of numerical detail about activity on that instant.
+To display an overlay of numerical detail about activity that occurred at a specific time, hover your mouse over the graph:
 
 image::monitoring-dashboards-hover.png[]
 
-
-If your server runs multiple applications, you can select applications in the list to view the performance of these in a separate set of charts. To do so, select the application and a side-panel will display on the right with this data.
+If your server runs multiple applications, you can view the performance of each in a separate set of charts. To do so, select the application checkboxes to display the details pane on the right with this data.
 
 === Server Metrics
 
-The graphs in this server's dashboard cover different metrics related to memory and CPU usage. You can see these metrics at a high level on the *Overview* tab, or you can view the *Breakdown* to see a detailed series of metrics that tracks each part of the memory separately.
+The graphs in this server dashboard provide metrics related to memory and CPU usage. You can see these metrics at a high level on the *Overview* tab or you can click *Breakdown* to see detailed memory usage metrics.
 
-The graphs in this dashboard display memory usage levels in both the *Overview* and the *Breakdown* tabs, and track these values:
+The graphs in this dashboard track these values:
 
-* *Used* - The amount of memory (in MB) that is currently in use. 
-If the Java VM attempts to increase the used memory to be greater than committed memory, even if the amount of used memory is increased to below the total memory, memory allocation fails.
-* *Committed* - The amount of memory (in MB) that's available for use by the Java VM. This amount varies over time. The amount of committed memory is always greater than or equal to the amount of used memory.
-* *Total* - The maximum amount of memory (in MB) for use by the server. The maximum amount can change or be undefined. 
+* *Usage*
++
+The amount of memory (in MB) currently in use. 
++
+If the Java VM attempts to increase the used memory to be greater than committed memory,  memory allocation fails even if the amount of used memory is increased to below the total memory.
+* *Total*
++
+The maximum amount of memory (in MB) for use by the server. The maximum amount can change or be undefined. 
+* *Committed*
++
+The amount of memory (in MB) available for use by the Java VM. This amount varies over time. The amount of committed memory is always greater than or equal to the amount of used memory.
 
 === JVM Heap Memory
```

Comment:
When I looked at the overall org., I was a little confused about how all the following peer topics are really peers to View Server Metrics and Server Metrics. Are each of the following actually one of the server metrics? Should these be grouped by whether they appear in Overview vs. Breakdown?